textproc/stanford-corenlp/Portfile

```diff
@@ -0,0 +1,255 @@
+                    set of stable and well-tested natural language \
+                    processing tools, widely used by various groups in \
+                    academia, industry, and government. The tools \
+                    variously use rule-based, probabilistic machine \
+                    learning, and deep learning components.
+
+homepage            http://stanfordnlp.github.io/CoreNLP/
+
+github.livecheck.regex \
+                    {([0-9.]+)}
+
+use_zip             yes
+github.tarball_from \
+                    archive
+
+checksums           rmd160  97f6bbcc3cf598251c36dbd2fcd8d51b6a6ac85d \
+                    sha256  cf46d0ee23410a5e2b0e43af33a4e7e42a958571f21479b3c0b305a13ce6c14a \
+                    size    74785152
+
+variant stanford_mirror \
+    description {Use the Stanford source mirror (slow).} {}
+
+if { [variant_isset "stanford_mirror"] } {
+    master_sites    http://nlp.stanford.edu/software
+    distname        ${name}-full-2018-10-05
+    checksums       rmd160  87842c9b14a41a7e8fb3f937a0633141354dc5f2 \
+                    sha256  833f0f5413a33e7fbc98aeddcb80eb0a55b672f67417b8d956ed9c39abe8d26c \
+                    size    393239982
+    livecheck.type  regex
+    livecheck.regex {version *([0-9.]+) *}
+}
+
+if {${name} eq ${subport}} {
+    depends_lib-append \
+                    port:apache-ant
+}
+if {${subport} ne "${name}-models"} {
+    depends_run-append \
+                    port:${name}-models
+}
+
+use_configure       no
+supported_archs     noarch
+
+# Required java version
+java.version        11+
+# JDK port to install if required java not found
+java.fallback       openjdk13
+
+set dest_java ${destroot}${prefix}/share/java/${name}
+set dest_doc ${destroot}${prefix}/share/doc/${name}
+
+if { ${name} eq ${subport} } {
+    if { ![variant_isset "stanford_mirror"] } {
+        build.cmd       ant
+        build.target
+        build.pre_args
+        build.args
+
+        post-build {
+            system -W ${worksrcpath}/classes \
+                "jar -cf ../${name}-${version}.jar edu"
+        }
+    } else {
+        build {}
+    }
+
+    proc xinstall_recursive_dir {srcbase destbase reldir} {
+        if [file isdirectory ${srcbase}/${reldir}] {
+            xinstall -m 0755 -d \
+                ${destbase}/${reldir}
+            fs-traverse f_or_d ${srcbase}/${reldir} {
+                set relative_path [strsed ${f_or_d} "s|${srcbase}||"]
+                if { [file isfile ${f_or_d}] } {
+                    xinstall -m 0644 \
+                        ${f_or_d} \
+                        ${destbase}${relative_path}
+                } elseif { [file isdirectory ${f_or_d}] } {
+                    xinstall -m 0755 -d \
+                        ${destbase}${relative_path}         
+                }
+            }
+        }
+    }
+
+    destroot {
+        # install jar files, scripts, classes, etc.
+        xinstall -m 0755 -d \
```

Comment:
Remove the redundant permissions spec `-m 0755`.

Reply:
Done—thanks.